### PR TITLE
feat: Inject sentry tracing information for rendered HTML

### PR DESCRIPTION
**Pull request by juliusknorr**
This allows to further trace from page rendering to network requests that are done in javascript.

https://docs.sentry.io/platforms/php/tracing/trace-propagation/custom-instrumentation/#inject-tracing-information-into-rendered-html

### CODE REVIEW

**Comment by juliushaertl** [comment]
Might need https://docs.sentry.io/platforms/javascript/tracing/trace-propagation/ as a second step

File: lib/AppInfo/Application.php [hunk 1]
```diff
@@ -76,6 +76,9 @@ public function boot(IBootContext $context): void {
 					'traces_sample_rate' => $config->getSamplingRate(),
 					'profiles_sample_rate' => $config->getProfilesSamplingRate(),
 				]);
+
+				\OC_Util::addHeader('meta', ['name' => 'baggage', 'content' => \Sentry\getBaggage()]);
```
Comment (ChristophWurst) [inline lib/AppInfo/Application.php:80]: private API :broken_heart:

Comment (juliushaertl) [reply lib/AppInfo/Application.php:80]: Yep didn't find a way through the public one, need to think a bit how that could be arranged

Comment (ChristophWurst) [reply lib/AppInfo/Application.php:80]: Can we use a global middleware?

Comment (juliushaertl) [reply lib/AppInfo/Application.php:80]: We cannot access the template instance from the template response in the middleware, but I have a local draft for extending that API, will check the next days.

Comment (juliusknorr) [reply lib/AppInfo/Application.php:80]: Seems I won't find time, let me close this until it becomes more relevant again or anyone else wants to pick up

**Pull request closed without merging**